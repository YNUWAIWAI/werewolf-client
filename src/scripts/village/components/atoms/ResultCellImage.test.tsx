/* global village */
import * as React from 'react'
import ResultCellImage from './ResultCellImage'
import {shallow} from 'enzyme'

describe('<ResultCellImage />', () => {
  test('alive', () => {
    const warpper = shallow(
      <ResultCellImage
        image="https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png"
        initial="A"
        status={village.AgentStatus.alive}
      />
    )

    expect(warpper.html()).toMatchSnapshot()
  })
  test('dead', () => {
    const warpper = shallow(
      <ResultCellImage
        image="https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png"
        initial="A"
        status={village.AgentStatus.dead}
      />
    )

    expect(warpper.html()).toMatchSnapshot()
  })
})
