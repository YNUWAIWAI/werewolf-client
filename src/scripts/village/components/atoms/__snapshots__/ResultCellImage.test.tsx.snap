// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultCellImage /> alive 1`] = `"<div class=\\"vi--result--cell--image \\"><div class=\\"vi--result--cell--image--initial\\">A</div><img class=\\"vi--result--cell--image--image\\" src=\\"https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png\\"/></div>"`;

exports[`<ResultCellImage /> dead 1`] = `"<div class=\\"vi--result--cell--image dead\\"><div class=\\"vi--result--cell--image--initial\\">A</div><img class=\\"vi--result--cell--image--image\\" src=\\"https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png\\"/></div>"`;

exports[`<ResultCellImage /> deathByAttack 1`] = `"<div class=\\"vi--result--cell--image dead\\"><div class=\\"vi--result--cell--image--initial\\">A</div><img class=\\"vi--result--cell--image--image\\" src=\\"https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png\\"/></div>"`;

exports[`<ResultCellImage /> deathByExecution 1`] = `"<div class=\\"vi--result--cell--image dead\\"><div class=\\"vi--result--cell--image--initial\\">A</div><img class=\\"vi--result--cell--image--image\\" src=\\"https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png\\"/></div>"`;

exports[`<ResultCellImage /> deathByFear 1`] = `"<div class=\\"vi--result--cell--image dead\\"><div class=\\"vi--result--cell--image--initial\\">A</div><img class=\\"vi--result--cell--image--image\\" src=\\"https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png\\"/></div>"`;

exports[`<ResultCellImage /> unnaturalDeath 1`] = `"<div class=\\"vi--result--cell--image dead\\"><div class=\\"vi--result--cell--image--initial\\">A</div><img class=\\"vi--result--cell--image--image\\" src=\\"https://werewolf.world/image/0.3/agent_icons/50x50/a_50x50.png\\"/></div>"`;
